azure-pipelines.yml

```diff
@@ -30,10 +30,10 @@ stages:
 
       dependencies:
         - source: '\\nirvana\Measurements\VeriStandAddons\instrument_addon_classes'
-          file: 'Protections.lvlibp'
+          file: '$target\Protections.lvlibp'
```

Comment:
Will not work until https://github.com/ni/niveristand-custom-device-build-tools/pull/163 is in.

Reply:
merged